projects/mpg123/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER kusano@email.com
```

Comment:
I guess this field isn't actually used, but did you leave a placeholder here intentionally?

Reply:
The MAINTAINER field? This is generated from `python infra/helper.py generate` (defaults to something like your@email.com). I've deleted it.

Reply:
Maybe we should get rid of that entirely.